.github/workflows/sonar.yaml

```diff
@@ -0,0 +1,17 @@
+# This is a template `sonar.yaml` file for ops charms
+# This file is managed by bootstack-charms-spec and should not be modified
+# within individual charm repos. https://launchpad.net/bootstack-charms-spec
+
+name: SonarCloud
+on:
+  workflow_run:
+    workflows:
+      - PR workflow running lint checkers, unit and functional tests
```

Comment:
This and line 17 needs to be changed to `Check workflow running linter, unit and functional tests` according to the name of workflow defined in check.yaml

Reply:
This should probably be fixed in a number of other places as well:

- https://github.com/canonical/charm-prometheus-blackbox-exporter/pull/6/files
- https://github.com/canonical/charm-duplicity/commit/9bd8835d65b4d4e286f772d9bd5412fe9f73c0dc
- https://github.com/canonical/charm-logrotated/commit/7f13e192ed232594d59ddec8818262f4d2a21c69

@jneo8